index.html

```diff
@@ -46,6 +46,207 @@ <h1>I'm Naan.
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <button class="buttons">
+                                <div class="see_project">See project</div>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card_works">
+                    <div class="snapshot_portfolio_1">
+                    </div>
+                    <div class="left_block underline">
+                        <div class ="primary_text">
+                            <div class="project_title "> Multi-Post Stories</div>
+                            <div class="sub_header">
+                                <div class="client">
+                                    <div class="client_text ">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text underline">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text underline">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
```

Comment:
- [ Required ] As requested by the previous reviewer please consider using a `ul` tag instead of the generic `div` tag, the `ul` tag is usually used to represent a list of elements on a webpage. It also makes your web page more accessible when a user is browsing your webpage using a screen reader, the screen reader will be to identify the list on the web page. Please consider refactoring this to `ul` tag as shown below.

```HTML
<ul class="tags">
     <li class="tag_1 tag_html">html</li>
     <li class="tag_2 tag_css">css</li>
     <li class="tag_3 tag_javascript">javascript</li>
  </ul>
```